src/devices/Hmc5883l/MeasuringMode.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Hmc5883l
+{
+    /// <summary>
+    /// The mode of HMC5883L measuring
+    /// </summary>
+    public enum MeasuringMode
+    {
+        Continuous = 0x00,
+        Single = 0x01,
+        Idle = 0x02
```

Comment:
if the device does nothing and does not allow reading when Idle is set then perhaps MeasuringMode should be settable?

Reply:
I'm just following the datasheet. I'll remove it.👌